schema/dereferenced-release-schema.json

```diff
@@ -28106,6 +28185,23 @@
             "description": "The single amendment object has been deprecated in favour of including amendments in an amendments (plural) array.",
             "deprecatedVersion": "1.1"
           }
+        },
+        "finalStatus": {
+          "title": "Contract final status",
+          "description": "The final status of the contract, from the closed [contractFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#contract-final-status) codelist.",
```

Comment:
```suggestion
          "description": "The final status of the contract, using the closed [contractFinalStatus](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#contract-final-status) codelist.",
```

Reply:
Done in https://github.com/open-contracting/standard/pull/1648/commits/4bab921b7905faf32c1a52101b733deb20ea7eb5